package.json

```diff
@@ -45,5 +45,8 @@
       "packages/*/{dist,test}/**/*.js",
       "packages/node_modules/**/*"
     ]
+  },
+  "dependencies": {
+    "gitbook-plugin-prism": "^2.3.0"
```

Comment:
is this supposed to be a devDep of the main repo?

Reply:
Man gitbook does this sometimes and I have no idea why! Makes me so mad